Geometry/HGCalMapping/test/HGCalMappingESSourceTester.cc

```diff
@@ -242,12 +238,13 @@ void HGCalMappingESSourceTester::analyze(const edm::Event& iEvent, const edm::Ev
   int zside(0), n_i(6000000);
   printf("[HGCalMappingIndexESSourceTester][produce]  Creating %d number of raw ElectronicsIds\n", n_i);
   uint32_t elecid(0);
-  auto start = now();
+  auto start = std::chrono::high_resolution_clock::now();
```

Comment:
`std::steady_clock` is better for measuring durations (`high_resolution_clock` is not guaranteed to be monotonic, i.e. it can go backwards)

Reply:
OK